app/controllers/evaluator_submission_assignments_controller.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+class EvaluatorSubmissionAssignmentsController < ApplicationController
+  before_action :set_challenge_and_phase
+  before_action :set_evaluator, only: [:index]
+  before_action :set_assignment, only: [:update]
+
+  def index
+    @evaluator_assignments = @phase.evaluator_submission_assignments.
+      includes(:submission, :evaluation).
+      where(user_id: @evaluator.id)
+    @assigned_submissions = @evaluator_assignments.
+      where(status: %i[completed in_progress not_started recused]).
+      ordered_by_status
+    @unassigned_submissions = @evaluator_assignments.
+      where(status: %i[unassigned recused_unassigned]).
+      ordered_by_status
+    @submissions_count = @assigned_submissions.group('evaluator_submission_assignments.status').count
+  end
+
+  # update only the status of the evaluation submission assignment to unassign or reassign an evaluator
+  def update
+    new_status = params[:status]&.to_sym
+
+    if update_assignment_status(new_status)
+      handle_successful_update(new_status)
+    else
+      handle_failed_update(new_status)
+    end
+  end
+
+  private
+
+  def set_challenge_and_phase
+    @phase = Phase.where(challenge: current_user.challenge_manager_challenges).find(params[:phase_id])
+    @challenge = @phase.challenge
+  end
+
+  def set_evaluator
+    @evaluator = params[:evaluator_id] ? User.find(params[:evaluator_id]) : current_user
```

Comment:
I don't think this should fallback to `current_user`, the request should fail if there is no evaluator specified.
The system should also restrict access to evaluators of the currently selected phase.
```suggestion
    @evaluator = @phase.evaluators.find(params[:evaluator_id])
```